app/controllers/concerns/foreman/controller/authentication.rb

```diff
@@ -92,7 +92,7 @@ def set_current_user(user)
   end
 
   def reset_oidc_session
-    backup_session_content([:sso_method, :expires_at]) { reset_session }
+    backup_session_content([:sso_method]) { reset_session }
```

Comment:
We can just
```suggestion
    backup_session_content { reset_session }
```

Reply:
@ezr-ondrej Perfect! that was a very nice catch :)